src/naive/date.rs

```diff
@@ -2027,6 +2029,14 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_date_from_num_days_from_ce_opt() {
+        let ndays_from_ce = |days| NaiveDate::from_num_days_from_ce_opt(days);
+        ndays_from_ce(i32::MAX);
```

Comment:
Can these two checks be moved into `test_date_from_num_days_from_ce` below?
Also, why not check the return value of the closure ?